CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [3.3.2-mintra] - 2023-12-12
+
+### Fixed
+- NinjectWebApiHttpApplication plugin can no longer be started multiple times on the same kernel.
```

Comment:
It can be started multiple times, but we have prevented registering the following bindings twice:
- DefaultFilterProviders
- DefaultModelValidatorProviders

Reply:
Details have been amended in [6a93507](https://github.com/mintra-as/Ninject.Web.WebApi/pull/1/commits/6a93507b142e9fba5bbb2f31ea7d7aed7870de20).